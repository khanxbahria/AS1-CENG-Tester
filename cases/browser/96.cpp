#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("pp", "jj","p");
Tab B = Tab("sss", "pp","ccc");
Tab C = Tab("ggg", "h","mm");
Tab D = Tab("h", "rr","yyy");
Tab E = Tab("d", "vvv","www");
Tab F = Tab("kk", "v","jj");
Tab G = Tab("qq", "ee","zzz");
Tab H = Tab("rrr", "nnn","i");
Tab I = Tab("n", "w","p");
Tab J = Tab("ww", "rrr","zzz");
Tab K = Tab("bb", "hhh","hh");
Tab L = Tab("kkk", "rrr","t");
Tab M = Tab("bb", "sss","k");
Tab N = Tab("kk", "w","vv");
Tab O = Tab("x", "xxx","qqq");
Tab P = Tab("nn", "yyy","f");
Tab Q = Tab("i", "ee","yyy");
Tab R = Tab("rrr", "jj","fff");
Tab S = Tab("ee", "o","hhh");
Tab T = Tab("tt", "c","iii");
Tab U = Tab("hhh", "dd","nn");
Tab V = Tab("hhh", "e","rrr");
Tab W = Tab("q", "ccc","aaa");
Tab X = Tab("ddd", "v","k");
Tab Y = Tab("hhh", "c","www");
Tab Z = Tab("uu", "ee","q");
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.newTab(A);
d.moveActiveTabTo(3);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.newTab(B);
d.changeActiveTabTo(0);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.newTab(C);
d.changeActiveTabTo(3);
a.changeActiveTabTo(1);
b.newTab(D);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.newTab(E);
b.changeActiveTabTo(4);
c.newTab(F);
d.changeActiveTabTo(4);
a.newTab(G);
b.changeActiveTabTo(4);
c.newTab(H);
d.moveActiveTabTo(4);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(I);
d.newTab(J);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.newTab(K);
d.changeActiveTabTo(4);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.newTab(L);
d.moveActiveTabTo(4);
a.newTab(M);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.newTab(N);
a.changeActiveTabTo(0);
b.newTab(O);
c.changeActiveTabTo(0);
d.newTab(P);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(Q);
d.changeActiveTabTo(1);
a.newTab(R);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(2);
b.newTab(S);
c.changeActiveTabTo(2);
d.newTab(T);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(U);
d.newTab(V);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.newTab(W);
a.moveActiveTabTo(0);
b.newTab(X);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(2);
b.newTab(Y);
c.moveActiveTabTo(2);
d.newTab(Z);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
d.moveActiveTabTo(0);
c.moveActiveTabTo(5);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
a.changeActiveTabTo(3);
d.changeActiveTabTo(3);
b.moveActiveTabTo(2);
a.changeActiveTabTo(0);
c.changeActiveTabTo(0);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
a.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
c.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
a.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(9);
brows->switchToWindow(4);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(3);
brows->print();
brows->newWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->newWindow();
brows->newWindow();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->getWindow(6);
brows->getWindow(5);
brows->closeEmptyWindows();
brows->switchToWindow(0);
brows->switchToWindow(3);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->closeWindow();
brows->getWindow(5);
brows->closeEmptyWindows();
brows->switchToWindow(0);
brows->getWindow(8);
brows->newWindow();
brows->getWindow(6);
brows->closeEmptyWindows();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->switchToWindow(8);
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(7);
brows->switchToWindow(0);
brows->closeWindow();
brows->closeWindow();
brows->print();
brows->getWindow(4);
brows->switchToWindow(7);
brows->getWindow(1);
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(2);
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(9);
brows->newWindow();
brows->getWindow(1);
brows->closeWindow();
brows->closeWindow();
brows->switchToWindow(6);
brows->newWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(7);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->getWindow(1);
brows->getWindow(9);
brows->newWindow();
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(6);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->print();
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->switchToWindow(4);
brows->getWindow(6);
brows->print();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->getWindow(2);
brows->getWindow(3);
brows->closeEmptyWindows();
brows->switchToWindow(1);
brows->switchToWindow(5);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
delete brows;
}
	return 0;
}