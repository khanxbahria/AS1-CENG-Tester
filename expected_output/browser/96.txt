The active tab is tab 31








ee
qq
zzz

jj
rrr
fff





vvv
d
www

sss
bb
k





xxx
x
qqq

o
ee
hhh

c
hhh
www

v
ddd
k

rr
h
yyy

jj
pp
p

w
n
p

nnn
rrr
i





dd
hhh
nn

v
kk
jj

ee
i
yyy

hhh
bb
hh

rrr
kkk
t

pp
sss
ccc

h
ggg
mm





e
hhh
rrr

ccc
q
aaa

rrr
ww
zzz

c
tt
iii

ee
uu
q

w
kk
vv

yyy
nn
f

The active tab is tab 36




























ee
qq
zzz

jj
rrr
fff





vvv
d
www

sss
bb
k





xxx
x
qqq

o
ee
hhh

c
hhh
www

v
ddd
k

rr
h
yyy

jj
pp
p

w
n
p

nnn
rrr
i





dd
hhh
nn

v
kk
jj

ee
i
yyy

hhh
bb
hh

rrr
kkk
t

pp
sss
ccc

h
ggg
mm





e
hhh
rrr

ccc
q
aaa

rrr
ww
zzz

c
tt
iii

ee
uu
q

w
kk
vv

yyy
nn
f

The active tab is tab 0




The active tab is tab 0




The browser is empty
The active tab is tab 1








The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 1








